packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.spec.js

```diff
@@ -123,77 +117,35 @@ describe('Component > ProjectAboutPage & Connector', function () {
           </Provider>
         </RouterContext.Provider>
       )
-      const heading = getByRole('heading', { name: 'About.PageHeading.title.team' })
+      const heading = screen.getByRole('heading', { name: 'About.PageHeading.title.team' })
       expect(heading).to.exist()
-      expect(getByText('No content yet.')).to.exist()
+      expect(screen.getByText('No content yet.')).to.exist()
     })
 
     it('should pass a navLinks array for pages with content', async function () {
-      const { getByRole } = render(
+      render(
         <RouterContext.Provider value={routerMock}>
           <Provider store={mockStore}>
             <Grommet theme={zooTheme} themeMode='light'>
-              <ProjectAboutPageConnector pageType='team' />
+              <ProjectAboutPageConnector pageType='results' />
             </Grommet>
           </Provider>
         </RouterContext.Provider>
       )
-      // AboutDropdown exists because default screen size is small
-      const dropdown = getByRole('button', { name: 'About.SidebarHeading' })
+      // AboutDropdown exists because default screen size is smaller than mobile breakpoint
+      const dropdown = screen.getByRole('button', { name: 'About.SidebarHeading' })
       expect(dropdown).to.exist()
       fireEvent.click(dropdown)
 
 			await waitFor(() => {
-				const navContainer = getByRole('navigation', { name: 'About.PageNav.title' })
-				const links = getAllByRole(navContainer, 'link')
+				const navContainer = screen.getAllByRole('navigation', { name: "About.PageNav.title"})
+				const links = getAllByRole(navContainer[0], 'link')
 				expect(links).to.have.lengthOf(3)
-			});
+			})
     })
   })
 
   describe('ProjectAboutPage', function () {
-    const aboutPageData = {
-      title: 'Title',
-      strings: {
-        title: 'Title',
-        content: 'This is some content.'
-      }
-    }
-
-    it('should render the dropdown nav on mobile screen sizes', function () {
```

Comment:
Deleted several of my old tests from this file because they're focused on styling or display based on viewport width, and it doesn't make a lot of sense to test that in a Node.js environment.